app/views/diary/steps/completed.html.erb

```diff
@@ -1,3 +1,5 @@
-Completion text goes here...
-<br><br>
-LINK TO INDEX
+<% content_for(:back_button) do %>
```

Comment:
have done this too...

Reply:
I'm happy to remove this change so that it doesn't clash 👍